w4af/core/controllers/dependency_check/platforms/ubuntu1204.py

```diff
@@ -32,10 +32,7 @@ class Ubuntu1204(Platform):
     PKG_MANAGER_CMD = 'sudo apt-get -y install'
     PIP_CMD = 'pip'
 
-    CORE_SYSTEM_PACKAGES = ['python3-pip', 'python3-dev',
```

Comment:
I'm not comfortable removing all these packages here. Some of them are required for the `pipenv install` to run, and if some of these dependencies are no longer necessary, we should look at which ones and remove the ones we don't need.

Removing them here is not a sustainable solution for the Docker API dependency check issue (see my comment on #630)